docs/self-managed/operational-guides/production-guide.md

```diff
@@ -0,0 +1,572 @@
+
+To get started, create two namespaces:
+
+```bash
+kubectl create namespace management
+kubectl create namespace orchestration
+```
+
+Within the `management` namespace, we will install Identity, Console, and all the Web Modeler components. Within the `orchestration` namespace, we will install the Camunda Orchestration Core component, along with Connectors and Optimize importer. We do not have to worry about installing each component separately since that will be handled by the Helm Chart automatically.
+
+#### Installing the Helm Chart
+
+The Camunda Helm Chart can be installed using the following command:
+
+```bash
+# This will install the latest Camunda Helm chart with the latest applications/dependencies.
+helm install camunda camunda/camunda-platform \
+    --values production-values.yaml
+```
+
+The next section will explain various configurations used in `production-values.yaml``
+
+### Ingress TLS Setup
+
+In order to access the Camunda Platform through HTTPS with ingress, you have to enable TLS. To do that, you require three things:
+
+1. A public registered domain that has configurable DNS records. In our example we will use `camunda.example.com` as the domain.
+2. A TLS certificate created for your domain. The certificate must be an X.509 certificate, issued by a trusted Certificate Authority. Also, the certificate must include the correct domain names (Common Name or Subject Alternative Names) to secure ingress resources. Please reach out to your DNS provider if you are unsure on how to create a TLS certificate. It is not recommended to use self-signed certificates.
+3. A TLS secret created from your TLS certificate. In our example, we will use a secret called `camunda-platform`. Please refer to the [kuberntes documentation](https://kubernetes.io/docs/concepts/configuration/secret/#tls-secrets) on how to make a TLS secret.
+
+Here is an example values.yaml configuration using the domain and TLS secret mentioned above:
+
+```yaml
+global:
+  ingress:
+    enabled: true
+    className: nginx
+    host: "camunda.example.com"
+    tls:
+      enabled: true
+      secretName: camunda-platform
+```
+
+Optionally, you can configure ingress for Zeebe GRP endpoints. Here is an example Zeebe GRPC Ingress setup:
+
+```yaml
+core:
+  ingress:
+    grpc:
+      enabled: true
+      className: nginx
+      host: "zeebe-grpc.camunda.example.com"
+      tls:
+        enabled: true
+        secretName: camunda-platform-core-grpc
+```
+
+For more information on the Ingress setup, please refer to our [ingress setup guide](/self-managed/setup/guides/ingress-setup.md)
+
+### Identity Provider Integration:
+
+Once secure HTTPS connections are enabled and correctly configured via Ingress, the next stage to consider is configuring authentication. In this example, we will use AWS Simple Active Directory. Here is an example configuration to add to your `production-values.yaml` file:
+
+```yaml
+global:
+  identity:
+    auth:
+      type: "MICROSOFT"
```

Comment:
I'm very confused. We're saying we're using `AWS Simple Active Directory,` but then everything links to Microsoft.

I don't think AWS would link to Microsoft but happy to learn more about it!
Coming to the next point, any guidance or help on where you got the values from?
Can we guide the user better to where they would get the `00000-000....` etc. etc. from? E.g. upstream AWS docs follow X, you receive Y.

- [ ] Why Microsoft if we say AWS?
- [ ] Can we guide the user better to where they should retrieve the values from?

Reply:
AWS Simple AD is just a managed instance of Microsoft AD on AWS.

Reply:
With regards to more guidance, I am hoping the customer clicks on the [OIDC guide](https://docs.camunda.io/docs/next/self-managed/setup/guides/connect-to-an-oidc-provider/#configuration) directly below the values.yaml

Reply:
The reason why I chose AWS Simple AD and not Azure entra was because I wanted to have everything on the same cloud provider. It doesn't make much sense to have  two cloud providers since nobody would really do that in an actually use-case.